RELEASING.md

```diff
@@ -33,39 +27,14 @@ The commit title should be `Release v<VERSION_NUMBER>` and the description shoul
 additions and deletions this versions will ship. This can be copied as is from the CHANGELOG file.
 Merge the PR back to main once it is approved.
 
-## 4.Create a draft Github Release <a href="step-4"></a>
-
-Go to the (project on Github)[https://github.com/signalfx/splunk-otel-js] and create a new release.
-On top of the release notes mention the version of OpenTelemetry API and other packages this
-version depends on. Do this for every release, even if the OpenTelemetry version has not changed.
-
-Next copy the changes, additions and deletions this version contains from the CHANGELOG file.
-
-Release name should be exact version number without the prefix `v` we use elsewhere.
-
-Save the release as a draft.
+## 4. Merge the PR <a href="step-4"></a>
 
+Once the PR is approved, you need to merge it. The rest of the process is automated. To start the release process, you need to push a signed tag `v<version>` to the internal Splunk mirror of this repository.
 
-## 5. Create and publish version git tag <a href="step-5"></a>
+## 5. Verify the newly released NPM package <a href="step-5"></a>
 
-Switch to `main` branch and pull the latest changes. Make sure your git head is on the release commit.
-Switch to the commit if it is not.
-
-Create a new git tag for the release with the `v` prefix. For example, if you're releasing `1.2.0`:
-
-```
-git tag v1.2.0
-git push origin v1.2.0
-```
-
-## 6. Verify CircleCI job and NPM package <a href="step-6"></a>
-
-Go to (the CircleCI project)[https://app.circleci.com/pipelines/github/signalfx/splunk-otel-js] and verify the build for your new version was successful.
+Go to the internal Splunk mirror of this repository and verify that the pipeline job for your new version was successful.
 
 Go to (https://www.npmjs.com/package/@splunk/otel)[https://www.npmjs.com/package/@splunk/otel] and verify the new package was published. It may take a few minutes for the npmjs.org web interface to reflect the new package but it should be installable instantly.
```

Comment:
```suggestion
Go to [https://www.npmjs.com/package/@splunk/otel](https://www.npmjs.com/package/@splunk/otel) and verify the new package was published. It may take a few minutes for the npmjs.com web interface to reflect the new package but it should be installable instantly.
```